model/src/main/scala/org/broadinstitute/dsde/rawls/model/Notification.scala

```diff
@@ -85,9 +85,9 @@ object Notifications {
     override val description = "Workspace Access Removed"
   })
 
-  case class WorkspaceInvitedNotification(recipientUserEmail: String, requesterId: String) extends Notification
+  case class WorkspaceInvitedNotification(recipientUserEmail: String, requesterId: String, workspaceName: WorkspaceName) extends Notification
```

Comment:
does this want to be an `Option`? is there a migration-y thing to do here if the pubsub queue has items in it once you make this change?